q2_deblur/tests/test_denoise.py

```diff
@@ -150,6 +150,13 @@ def test_with_stats(self):
         _, _, obs_stats = denoise_16S(self.demux_seqs, 100, sample_stats=True)
         pdt.assert_frame_equal(obs_stats, exp_stats)
 
+    def test_with_underscore_in_id(self):
+        bad_seqs = SingleLanePerSampleSingleEndFastqDirFmt(
+            self.get_data_path('sample_seqs_16S_underscore'), 'r')
+        with self.assertRaisesRegex(
+                ValueError, 'Deblur cannot.*L3S_313.'):
+            denoise_16S(bad_seqs, 100, sample_stats=True)
```

Comment:
```suggestion
            denoise_16S(bad_seqs, 100)
```

No need to include that flag, since we aren't testing that behavior.